### PR TITLE
updated animating time series for QGIS 3.16 and higher

### PR DESCRIPTION
**Pull request by Santhosh-M31**

### CODE REVIEW

File: source/docs/3/animating_time_series.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
-Animating Time Series Data (QGIS3)
-==================================
+Animating Time Series Data (QGIS 3.16)
```
Comment (spatialthoughts) [inline source/docs/3/animating_time_series.rst:1]: Keep it QGIS3 only.

File: source/docs/3/animating_time_series.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+======================================
 
-Time is an important component of many spatial datasets. Along with location information, time providers another dimension for analysis and visualization of data. If you are working with dataset that contains timestamps or have observations recorded at multiple time-steps, you can easily visualize it using the **TimeManager** plugin in QGIS. TimeManager allows you to view and export 
+Time is an important component of many spatial datasets. Along with location information, time providers another dimension for analysis and visualization of data. If you are working with dataset that contains timestamps or have observations recorded at multiple time-steps, you can easily visualize it using the **Temporal controler** only available from version 3.16. Temporal controler allows you to view and export
```
Comment (spatialthoughts) [inline source/docs/3/animating_time_series.rst:4]: QGIS 3.14 onwards

File: source/docs/3/animating_time_series.rst [hunk 3]
```diff
@@ -61,101 +54,64 @@ Procedure
...
       :align: center
 
-8. The source data contains an attribute ``dateofocc`` - representing the date on which the incident took place. This is the field that will be used by the plugin to determine the points that are rendered for each time period. Select ``ASAM_events`` as the :guilabel:`Layer` and ``dateofocc`` as the :guilabel:`Start time`. The :guilabel:`End time` should be set to ``Same as start``. Click :guilabel:`OK`.
+8. Now a clock symbol will appear next to the layer name. Click on the ``Temporal Control Panel`` (Clock icon) from Map Navigation Toolbar.
 
   .. image:: /static/3/animating_time_series/images/8.png
       :align: center
 
-9. Back in the :guilabel:`Time manager settings` window, click :guilabel:`OK`.
+9. Click on the ``Animated Temporal Navigation`` (play icon) to start the rendering. To increase the duration click ``Temporal Settings`` (yellow gear icon) and increase the input frame rate (frames per second).
```
Comment (spatialthoughts) [inline source/docs/3/animating_time_series.rst:77]: Change rendering to animation

File: source/docs/3/animating_time_series.rst [hunk 4]
```diff
@@ -61,101 +54,64 @@ Procedure
...
+11. Click the checkbox to enable it and click ``Insert an Expression`` button and enter the following expression to display the year.
 
-.. note::
+.. code-block:: none
 
-  As TimeManager works by applying a filter on the layer, it only works with layer types that support this feature. Most data source types do support it - with a notable exception being temporary memory layers. If you had done some processing earlier and have a temporary layer, right-click and select :guilabel:`Make Permanent` before using TimeManager on that layer.
-  
-11. Now you are ready to see the animation. Click the :guilabel:`Play` button to see the yearly piracy hotspot animation.
+  format_date(@map_start_time, 'yyyy-MM-dd')
+.
```
Comment (spatialthoughts) [inline source/docs/3/animating_time_series.rst:92]: remove extra '.'

File: source/docs/3/animating_time_series.rst [hunk 5]
```diff
@@ -61,101 +54,64 @@ Procedure
...
 
-  .. image:: /static/3/animating_time_series/images/11.gif
-      :align: center
+  .. image:: /static/3/animating_time_series/images/11.png
+      :align: center 
 
-12. You will notice that for each frame of the animation, a date is displayed at the bottom-right. Instead of the full date and time, let's change it to display the year that the map represents. Click :guilabel:`Settings` in the :guilabel:`Time Manager` panel. Click :guilabel:`Time display options` in the :guilabel:`Time manager settings` dialog.
+12. Select :guilabel:`font size` as 25, set :guilabel:`background bar colour` as ``White`` and set the transparency to ``50%``. In :guilabel:`Placement` choose ``Bottom Right``. Now click Ok.
```
Comment (spatialthoughts) [inline source/docs/3/animating_time_series.rst:97]: put 25  in ``25``.

File: source/docs/3/animating_time_series.rst (hunk 1 above)
Comment (Santhosh-M31) [reply source/docs/3/animating_time_series.rst]: Changed

File: source/docs/3/animating_time_series.rst (hunk 2 above)
Comment (Santhosh-M31) [reply source/docs/3/animating_time_series.rst]: changed

File: source/docs/3/animating_time_series.rst (hunk 3 above)
Comment (Santhosh-M31) [reply source/docs/3/animating_time_series.rst]: changed

File: source/docs/3/animating_time_series.rst (hunk 4 above)
Comment (Santhosh-M31) [reply source/docs/3/animating_time_series.rst:92]: Need help

File: source/docs/3/animating_time_series.rst (hunk 5 above)
Comment (Santhosh-M31) [reply source/docs/3/animating_time_series.rst]: changed

**Pull request merged**